app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,73 @@
+body {
+  background-color: #fff;
+  color: #333;
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
```

Comment:
`font-size` for `html` and `body` should always be set to `100%`, to ensure responsive compatibility